coderd/workspaceagents.go

```diff
@@ -1601,6 +1601,7 @@ func convertWorkspaceAgentMetadata(db []database.WorkspaceAgentMetadatum) []code
 	result := make([]codersdk.WorkspaceAgentMetadata, len(db))
 	for i, datum := range db {
 		result[i] = codersdk.WorkspaceAgentMetadata{
+			Order: datum.DisplayOrder,
```

Comment:
How do I know if this value is always returned from `datum.DisplayOrder`? Does Go return `0` if the user didn't define it in the terraform config?